### ISSUE TITLE
UselessParentheses does not detected some cases in switch/case

### ISSUE THREAD

**Issue by carusogabriel** [slevomat/coding-standard#416]
```php
switch (true) {
    case ($this->foo === null):
        // these parentheses **are not** detected
    case (empty($this->bar)):
       // these parentheses **are** detected
}
```

While we are here, I believe there's a change in the `SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly.ReferenceViaFullyQualifiedName` sniff that now detects one level FQN classes missing in the release notes.

**Comment by kukulich** [comment]
> While we are here, I believe there's a change in the SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly.ReferenceViaFullyQualifiedName sniff that now detects one level FQN classes missing in the release notes.

I don’t understand :(

**Comment by kukulich** [comment]
The `case` bug is fixed in https://github.com/slevomat/coding-standard/commit/bd0ee024a86fc723ba04e77f1c24d583665be461
